features/dd-sdk-android-session-replay/api/apiSurface

```diff
@@ -22,12 +22,26 @@ interface com.datadog.android.sessionreplay.internal.recorder.OptionSelectorDete
   fun isOptionSelector(android.view.ViewGroup): Boolean
 data class com.datadog.android.sessionreplay.internal.recorder.SystemInformation
   constructor(GlobalBounds, Int = Configuration.ORIENTATION_UNDEFINED, Float, String? = null)
-abstract class com.datadog.android.sessionreplay.internal.recorder.mapper.BaseWireframeMapper<T: android.view.View, S: com.datadog.android.sessionreplay.model.MobileSegment.Wireframe> : WireframeMapper<T, S>
-  constructor(com.datadog.android.sessionreplay.utils.StringUtils = StringUtils, com.datadog.android.sessionreplay.utils.ViewUtils = ViewUtils)
+class com.datadog.android.sessionreplay.internal.recorder.base64.Base64Serializer
+interface com.datadog.android.sessionreplay.internal.recorder.base64.ImageCompression
+  fun getMimeType(): String?
+  fun compressBitmapToStream(android.graphics.Bitmap): java.io.ByteArrayOutputStream
+class com.datadog.android.sessionreplay.internal.recorder.base64.WebPImageCompression : ImageCompression
+  override fun getMimeType(): String?
+  override fun compressBitmapToStream(android.graphics.Bitmap): java.io.ByteArrayOutputStream
+  companion object 
```

Comment:
probably shouldn't be here, but it is not important

Reply:
it is, it is exposed by the `BaseWireframeMapper`. We took the decision to push it now and see how we can hide this better later. Not very important for now.